app/src/main/java/edu/grinnell/grinnell_publications_android/Services/Implementation/RealmLocalClient.java

```diff
@@ -49,22 +71,43 @@ public void savePublication(Publication publication) {
 
     @Override
     public List<Publication> getAllPublications() {
-        return null;
+        Realm realm = Realm.getDefaultInstance();
+        RealmResults<RealmPublication> pubResults = realm.where(RealmPublication.class).findAll();
+        List<Publication> mResults = new ArrayList<>();
+        mResults.addAll(pubResults.subList(0, pubResults.size()));
```

Comment:
+1 to that. Also, what is this "0" magic number here? Make a constant with a descriptive variable name.